### ISSUE TITLE
high cpu consumption

### ISSUE THREAD

**Issue by archercreat** [TimUntersberger/wwm#300]
**Version (which release did you use?)**
Latest
**Describe the bug**
There seems to be 1 thread that constantly consumes 5% on my cpu
![Untitled](https://user-images.githubusercontent.com/41573188/168480217-effce064-7eae-4801-a9cd-06c19ee380d7.png)

```
.\download_release.ps1 master-release
start-process $("$env:APPDATA\nog\bin\nog.exe")
```

**Expected behavior**
Compared to other tilling managers for windows this should not be happening.

**Actual behavior**

**Comment by TimUntersberger** [comment]
Is the cpu usage increasing over time?

**Comment by archercreat** [comment]
No, it stays 5% constantly. Looks like an infinite loop to me.
I've made a simple program that also consumes 5% on my cpu.
```cpp
int main(int argc, char** argv)
{
	volatile int i = 0;
	while (1) {
		i++;
	}
}
```

![image](https://user-images.githubusercontent.com/41573188/168555978-1d9a8347-41d3-4805-b7a8-90356b090190.png)

**Comment by TimUntersberger** [comment]
Well nog is using an infinite loop. Did the cpu usage increase after a windows update or has it been the same for you?

**Comment by archercreat** [comment]
No, i've just tried it recently and thought it was unusual that 1 thread consumes all cpu time

**Comment by TimUntersberger** [comment]
>all cpu time

You mean 5% right?

**Comment by archercreat** [comment]
> > all cpu time
> 
> You mean 5% right?

Yea I’ve noticed it because cpu fans went crazy

**Comment by archercreat** [comment]
I have `i9-10850K` which has 10 cores 20 threads. So 5% cpu = 1 thread

**Comment by michaelrommel** [comment]
I have the same issue. Just downloaded nog, started it, activated work mode, fans spun up. It is unfortunately unusable in this state. From what I read, I really liked the minimal approach.

**Comment by GuiReboucas** [comment]
> I have the same issue. Just downloaded nog, started it, activated work mode, fans spun up. It is unfortunately unusable in this state. From what I read, I really liked the minimal approach.

Also had the same problem
